### PR TITLE
Add possibility to style Google Maps

### LINKED ISSUE #20: Add possibility to change Google Maps style

### PR DESCRIPTION
**Pull request by szdziedzic**

### CODE REVIEW

File: expo-maps/android/src/main/java/expo/modules/maps/GoogleMapsView.kt [hunk 1]
```diff
@@ -45,6 +46,18 @@ class GoogleMapsView(context: Context): LinearLayout(context), OnMapReadyCallbac
     }
   }
 
+  fun setMapStyle(jsonStyleString: String) {
+    if (jsonStyleString.isNotEmpty()) {
+      updateMap {
+        this.googleMap.setMapStyle(MapStyleOptions(jsonStyleString))
```
Comment (Zwatotem) [inline expo-maps/android/src/main/java/expo/modules/maps/GoogleMapsView.kt:52]: Seems like you forgot to handle the exception here, like you did for iOS. Or is there a specific reason for that?

**Review by Zwatotem — COMMENTED** [review]
Other that this one comment above everything looks and works right.

File: expo-maps/android/src/main/java/expo/modules/maps/GoogleMapsView.kt (hunk 1 above)
Comment (szdziedzic) [reply expo-maps/android/src/main/java/expo/modules/maps/GoogleMapsView.kt:52]: It doesn’t look like `MapStyleOptions` constructor throws an error on Android as `GMSStyle` one does on iOS. Please correct me if I’m wrong.

https://developers.google.com/android/reference/com/google/android/gms/maps/model/MapStyleOptions#public-constructors
Zrzut ekranu 2022-01-22 o 23 05 00

**Review by p-karcz — APPROVED** [review]

**Review by ellwiii — APPROVED** [review]

**Review by lukmccall — APPROVED** [review]
LGTM 🚀

**Pull request merged**